deploy/docker/seqr/Dockerfile

```diff
@@ -1,25 +1,30 @@
 FROM google/cloud-sdk:335.0.0
 
-ARG SEQR_REPO=https://github.com/populationgenomics/seqr
-ARG SEQR_GIT_SHA
 ARG SEQR_SERVICE_PORT
 ENV SEQR_SERVICE_PORT=$SEQR_SERVICE_PORT
 ENV PATH $PATH:/root/google-cloud-sdk/bin
 EXPOSE $SEQR_SERVICE_PORT
 
+WORKDIR /opt/seqr
+ADD requirements.txt /opt/seqr/
```

Comment:
I think `COPY` is more common to use than `ADD` for simple files?

Reply:
Yep, makes sense. Changed ADD to COPY.